assets/js/components/game-state.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+
+class GameState extends React.Component {
```

Comment:
Doesn’t look like this maintains any internal state either, could this potentially be a functional / stateless component?